FirebaseAuth/Sources/Swift/AuthProvider/GameCenterAuthProvider.swift

```diff
@@ -72,8 +71,7 @@
                https://developer.apple.com/documentation/gamekit/gkplayer
                **/
               let displayName = localPlayer.alias
-              let credential = GameCenterAuthCredential(withPlayerID: localPlayer.playerID,
```

Comment:
Seems we can't deprecate the playerID for now. The reason is it's required by our backend together with the credentials generated by `generateIdentityVerificationSignature`, which is deprecated after 13.5.

I suggest we remove playerID together with the code block from L63-L85 after iOS 13.5.

Reply:
@ryanwilson : Thoughts about bumping to a minimum iOS 13.5, etc instead of 13.0 in Firebase 11

Reply:
A minimum of 13.5 would probably cause too much confusion, so lets wait until we go to at least 14. I'll abandon this PR for now. thanks.